owasp.suppressions.xml

```diff
@@ -1,40 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <suppress until="2024-05-03Z">
-        <notes>
-            <![CDATA[file name: jackson-databind-2.13.5.jar]]>
-
-            This vulnerability can cause an OOM error when trying to serialize an object that contains cyclic
-            dependencies. This is not a legitimate attack vector since the object must be crafted in memory by the
-            application itself; this can only arise as a result of developer error and would be caught by automated
-            testing.
-        </notes>
-        <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
-        <cve>CVE-2023-35116</cve>
-    </suppress>
-    <suppress until="2024-05-03Z">
-        <notes>
-            <![CDATA[file name: jose4j-0.7.7.jar]]>
-
-            Jose is a transitive dependency of the Notify API client. This vulnerability relates to a denial-of-service
-            attack when Jose is asked to mint a JWT using a large iteration count. Since Jose is only used by the Notify
-            client, and does not receive external input, this is not a risk for us.
-        </notes>
-        <packageUrl regex="true">^pkg:maven/org\.bitbucket\.b_c/jose4j@.*$</packageUrl>
-        <vulnerabilityName>CVE-2023-51775</vulnerabilityName>
-    </suppress>
-    <suppress until="2024-05-03Z">
-        <notes>
-            <![CDATA[file name: jose4j-0.7.7.jar]]>
-
-            Jose is a transitive dependency of the Notify API client. This vulnerability relates to a weak algorithm
-            which is allowed by Jose. Since Jose is only used by the Notify client, and does not receive external input,
-            this is not a risk for us.
-        </notes>
-        <packageUrl regex="true">^pkg:maven/org\.bitbucket\.b_c/jose4j@.*$</packageUrl>
-        <vulnerabilityName>CVE-2023-31582</vulnerabilityName>
-    </suppress>
-    <suppress until="2024-05-03Z">
+    <suppress until="2024-12-01Z">
```

Comment:
Why this particular date?

Reply:
I've made a ticket for December Vulnerability patching, and will include a note on that ticket to check if the PR has been merged and postpone further if not.